registry/registry_db.go

```diff
@@ -64,8 +64,8 @@ type AdminMetadata struct {
 	UserID                string             `json:"user_id" post:"exclude"` // "sub" claim of user JWT who requested registration
 	Description           string             `json:"description"`
 	SiteName              string             `json:"site_name"`
-	Institution           string             `json:"institution" validate:"required"` // the unique identifier of the institution
-	SecurityContactUserID string             `json:"security_contact_user_id"`        // "sub" claim of user who is responsible for taking security concern
+	Institution           string             `json:"institution" validate:"required"`                                                                                // the unique identifier of the institution
```

Comment:
It would be useful if you gave an example of what the unique identifier looks like in this comment. Is it a string name? Is it a random number? Is it a hash?

Reply:
The type of the value really depends on whatever provided by `Registry.Institutions` or `Registry.InstitutionsUrl`, so I updated the comment to reflect this.